Composer/plugins/azurePublish/src/schema.ts

```diff
@@ -125,9 +125,9 @@ const schema: JSONSchema7 = {
       },
       luis: {
         authoringKey: '<authoring key>',
-        authoringEndpoint: '<luis authoring endpoint>',
+        authoringEndpoint: '',
```

Comment:
because when user don't input a value, the placeholder will be set as a value. And in the server it will not use the default endpoint, and it will trigger a absolute path error.